src/chapters/classes/class-methods.rst

```diff
@@ -0,0 +1,102 @@
+.. _adding-class-methods:
+
+Assigning Class Methods
+========================
+
+Just as with objects, we may want to add methods to our classes in addition to properties. 
+Previously, we learned how to use constructors in class declarations to set the values of the properties.
+When working with assigning methods in classes, we can either create them `outside` or `inside` in the ``constructor``.
+
+Assigning Methods Outside ``constructor``
+-----------------------------------------
+
+When assigning methods outside of the ``constructor``, we simply declare our methods the same way that we would normally do for objects.
```

Comment:
We should sync up on this! I don't think adding a link would be helpful because that comparison with examples has been made in the chapter before.

Reply:
True, but it is a good practice to provide a link to the earlier chapters. 

This is for the students who need a quick reminder. The internal link provides them with a quick and easy path to follow, rather than forcing them to visit the TOC and wondering, "Which section(s) hold the information I want?"